ethcore/sync/src/block_sync.rs

```diff
@@ -231,6 +231,19 @@ impl BlockDownloader {
 		self.state = State::Blocks;
 	}
 
+	/// Reset sync to the specified block
+	pub fn reset_to_block(&mut self, start_hash: &H256, start_number: BlockNumber) {
```

Comment:
```suggestion
	fn reset_to_block(&mut self, start_hash: &H256, start_number: BlockNumber) {
```
does it have to be public?

Reply:
Ofc no ;-)